.github/workflows/autobuild.yml

```diff
@@ -48,6 +48,8 @@ jobs:
          # Checkout code
          - name:                    Checkout code
            uses:                    actions/checkout@v2
+         - name: Checkout portaudio submodule
+           run: git submodule update --init -- libs/portaudio
```

Comment:
the first action does not need submodules, please remove these lines

(in the second action its needed, and included by the "with:   submodules:   true" of the checkout